src/core/Microsoft.PowerToys.Settings.UI/ViewModels/FancyZonesViewModel.cs

```diff
@@ -112,7 +112,44 @@ public bool IsEnabled
                     OutGoingGeneralSettings snd = new OutGoingGeneralSettings(generalSettings);
 
                     ShellPage.DefaultSndMSGCallback(snd.ToString());
-                    OnPropertyChanged("IsEnabled");
+                    OnPropertyChanged(nameof(IsEnabled));
+                    OnPropertyChanged(nameof(TextColor));
+                }
+            }
+        }
+
+        public string TextColor
+        {
+            get
+            {
+                if (this.IsEnabled)
+                {
+                    GeneralSettings generalSettings = SettingsUtils.GetSettings<GeneralSettings>(string.Empty);
+
+                    var defaultTheme = new Windows.UI.ViewManagement.UISettings();
+
+                    var uiTheme = defaultTheme.GetColorValue(Windows.UI.ViewManagement.UIColorType.Background).ToString();
+
+                    if (generalSettings.Theme.ToLower() == "dark")
+                    {
+                        return "#FFFFFFFF";
+                    }
+                    else if (generalSettings.Theme.ToLower() == "light")
+                    {
+                        return "#FF000000";
+                    }
+                    else if (generalSettings.Theme.ToLower() == "system" && uiTheme == "#FF000000")
+                    {
+                        return "#FFFFFFFF";
+                    }
+                    else
+                    {
+                        return "#FF000000";
+                    }
+                }
+                else
+                {
+                    return "#FF7A7A7A";
```

Comment:
Can we return Brushes instead of hard coded colors? I think we have these defined in one of the styles XAML files.

Or, we can just leverage the build in UWP brushes.

Reply:
Agreed. More so this is WinUI so please use built in brushes.

Reply:
Actually why is this not just a general built in brush. If it is, would handle light, dark and high contrast by default. Why are we manually doing the check.

Reply:
I tried the brushes, they were not adapting to changing themes. But I will do a double check again.

Reply:
what brush did you use?  one of the theme resources?  https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources

Reply:
Yes: 
![image](https://user-images.githubusercontent.com/58791731/86952286-9c557700-c107-11ea-9123-ab4ace5bb54e.png)